src/components/VrxDateRangePicker/VrxDateRangePicker.vue

```diff
@@ -39,37 +41,42 @@
           :helper-text="(helperTextStop ? helperTextStop : (helperTextStart ? '&nbsp;' : undefined))"
       />
     </div>
-    <div
-        v-if="showDropdown"
-        class="dropdown-vrx-picker absolute w-48 mt-12 p-4 h-auto text-sm rounded-lg bg-gray-50 dark:bg-gray-700 flex flex-col gap-2 shadow-md"
-        tabindex="-1"
-        @focusout="showDropdown = false"
-    >
-      <div class="flex flex-row justify-between items-center w-full">
-        <VrxIcon icon="chevron-left" class="rounded-lg hover:bg-gray-200 dark:hover:bg-gray-500 cursor-pointer"
-                 @click="changeType(-1)"/>
-        <span class="rounded-lg font-bold pt-1 pb-1 pr-2 pl-2 select-none">{{ type }}</span>
-        <VrxIcon icon="chevron-right" class="rounded-lg hover:bg-gray-200 dark:hover:bg-gray-500 cursor-pointer"
-                 @click="changeType(+1)"/>
-      </div>
-      <div class="flex flex-col justify-center w-full">
-        <div
-            v-for="button in currentButtons"
-            @click="button.action()"
-            class="rounded-lg hover:bg-gray-200 dark:hover:bg-gray-500 pt-1 pb-1 pr-2 pl-2 cursor-pointer justify-center flex select-none hover:text-blue-700 dark:hover:text-white">
-          {{ button.value }}
+  </div>
+    <div class="fixed top-0 left-0 w-full h-full z-20" v-if="showDropdown">
```

Comment:
riesci a mandarmi in privato il tuo sito storybook ? voglio capire come rende centrato sullo schermo